docs/dev-guide/using-sharrow.md

```diff
@@ -0,0 +1,413 @@
+
+# Using Sharrow
+
+This page will walk through an exercise of running a model with `sharrow`.
+
+## How it Works
+
+Sharrow accelerates ActivitySim in part by using numba to create optimized and
+pre-compiled versions of utility specification files, and caching those bits
+of code to disk.
+
+```{important}
+Running the compiler needs to be done in single-process mode, otherwise the
+various process all do the compiling and compete to write to the same cache
+location on disk, which is likely to fail.  You can safely run in
+multiprocessing mode after all the compilation for all model components is
+complete.
+```
+
+## Getting the Code
+
+We'll assume that activitysim and sharrow have been installed in editable
+mode, per the [developer install instructions](install.md).
+
+The code to implement `sharrow` in `activitysim` is in the
+[sharrow-black](https://github.com/camsys/activitysim/tree/sharrow-black) branch
+of the [camsys/activitysim](https://github.com/camsys/activitysim) repository.
+If you have checked out from this repository directly, you can just switch to
+the correct branch:
+
+```sh
+cd activitysim
+git switch sharrow-black
+cd ..
+```
+
+If your local Git repository is configured with only the main ActivitySim
+parent repo as its remote, you can find the code in
+[PR #579](https://github.com/ActivitySim/activitysim/pull/579), accessible on the
```

Comment:
Update this to the new PR. When it's merged, remember to update this part again.

Reply:
Since I expect we will be transitioning from PR to merged into the develop branch and then to an upcoming release relatively promptly, I am just removing this section of instructions entirely; users will need no special instructions for how to access the code because it will just be in ActivitySim without any extra steps.